website/docs/k8s-support.md

```diff
@@ -0,0 +1,45 @@
+---
+id: k8s-support
+title: Kubernetes Support
+sidebar_label: Kubernetes Support
+original_id: k8s-support
+---
+
+---
+
+### Control Plane Resource Requirements
+
+The Resource requests and limits provided here have been calculated using data gathered by manually using a command `kubectl top` and Vertical-Pod-Autoscaler & also through recommendation by a great utility Goldilocks.The resources are getting monitored continuously and the information below will be updated as the metrics changes.
```

Comment:
We can restructure this a bit to provide more detail on the resource req estimation process.